examples/kubernetes/cilium-ds-gsg.yaml

```diff
@@ -0,0 +1,198 @@
+kind: ClusterRole
```

Comment:
I added this file to be used during the k8s-gsg-test.sh as we cannot use etcd as the key-value store for Cilium in the GSG test because the test is using etcd version 3.0.17, and there currently is no image `>= 3.1.0` for etcd in the container registry used by kubeadm for [etcd](https://console.cloud.google.com/gcr/images/google-containers/GLOBAL/etcd-amd64)